src/controllers/organizationController.ts

```diff
@@ -65,8 +72,52 @@ export default class OrganizationController {
       }
       return withValidatedOrValidationError(
         OrganizationRegistrationParams.decode(req.body),
-        (organizationRegistrationParams: OrganizationRegistrationParams) =>
-          registerOrganization(organizationRegistrationParams, user)
+        async (
+          organizationRegistrationParams: OrganizationRegistrationParams
+        ) => {
+          const errorResponseOrSuccessResponse = await registerOrganization(
+            organizationRegistrationParams,
+            user
+          );
+          return errorResponseOrSuccessResponse.map(async response => {
+            const organization = response.payload;
+            const outputFolder = `./documents/${organization.ipa_code}`;
+            try {
+              await fs.promises.mkdir(outputFolder, { recursive: true });
+              const arrayOfMaybeError = await Promise.all([
+                this.documentService.generateDocument(
+                  localeIt.organizationController.registerOrganization.contract.replace(
+                    "%s",
+                    `${organization.name} ${organization.fiscal_code}`
+                  ),
+                  `${outputFolder}/contract.pdf`
+                ),
+                this.documentService.generateDocument(
+                  localeIt.organizationController.registerOrganization.delegation
+                    .replace(
```

Comment:
with another PR, consider refactoring this using https://www.i18next.com/translation-function/interpolation templating (add a todo comment linked to a pivotal task suffices)